Dockerfile

```
@@ -143,5 +143,9 @@ RUN apt-get update -qq && apt-get -y --no-install-recommends install \
 # Check to make sure the binaries are available by loading the bedr library
 RUN Rscript -e "library(bedr)"
 
+# packages required for collapsing RNA-seq data by removing duplicated gene symbols
+RUN R -e "install.packages(c('knitr', 'DT'), dependencies = TRUE, repos = 'http://cran.rstudio.com/')"
```

`knitr` should already be installed, and we would prefer to keep packages installed from the docker set default repository where possible to improve reproducibility. 
```suggestion
RUN R -e "install.packages('DT', dependencies = TRUE)"
```